features/bootstrap/FeatureContext.php

```diff
@@ -103,7 +101,7 @@ public function isInstalled($command)
 	}
 
 	/**
-	 * @When I run :command
+	 * @When /I run '(.*)'|"(.*)"/
```

Comment:
I think this whole regex needs to be wrapped in parenthesis and the inner parenthesis are not required.
```
/I run ('.*'|".*")/
```

Reply:
This regex does not work as it also matches the outer quotes.

Reply:
Let's go with the one already added.
```
@When /I run '(.*)'|"(.*)"/
```